mne/_fiff/meas_info.py

```diff
@@ -3729,7 +3729,10 @@ def _ensure_infos_match(info1, info2, name, *, on_mismatch="raise"):
     if (info1["dev_head_t"] is None) != (info2["dev_head_t"] is None) or (
```

Comment:
passing comment / nitpick: to me xor infix (`^`) seems clearer than `!=` here

```suggestion
    if (info1["dev_head_t"] is None) ^ (info2["dev_head_t"] is None) or (
```